shared/Utils.ts

```diff
@@ -8,3 +8,10 @@ export const getVCsOrderedByPinStatus = (vcMetadatas: VCMetadata[]) => {
   );
   return pinned.concat(unpinned);
 };
+
+export enum FlowType {
```

Comment:
ReName it to more specfic flow : VCShareFlowType

Reply:
done